sirfilecache.c

```diff
@@ -34,17 +34,13 @@ sirfileid _sir_addfile(const char* path, sir_levels levels, sir_options opts) {
     if (!_sir_sanity())
         return NULL;
 
-    sirfcache* sfc = _sir_locksection(SIRMI_FILECACHE);
-    if (!sfc) {
-        _sir_seterror(_SIR_E_INTERNAL);
-        return NULL;
-    }
+    _SIR_LOCK_SECTION(sirfcache, sfc, SIRMI_FILECACHE, 0);
```

Comment:
This is 0 instead of NULL in anticipation of #158 being merged.

Reply:
+1